mirdata/datasets/indexes/giantsteps_key_index_1.0.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "+",
+  "version": "1.0",
```

Comment:
Does this change match the version in Zenodo?

Reply:
whoops you're right! Fixing it